yarn-project/circuits.js/src/tests/factories.ts

```diff
@@ -433,10 +433,22 @@ export function makePrivateKernelTailCircuitPublicInputs(seed = 1, full = true):
  * @returns Public call request.
  */
 export function makePublicCallRequest(seed = 1): PublicCallRequest {
+  const childCallContext = makeCallContext(seed + 0x2, makeAztecAddress(seed));
+  const parentCallContext = CallContext.from({
```

Comment:
Actually I discussed this with @sirasistant and ended up agreeing with him: since the parent and the child have to be related, this is more explicit than just calling `makeCallContext` with some magic seeds